_docs/extending-expired-ca-cert.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Extending Expired Puppet CA Certificate
+date: 2025-08-06
+summary: How to extend the expiration date of the Puppet CA certificate
+---
+
+## Using a Bolt Module
+
+If you're using Bolt in your environment, there's a
+[ca_extend](https://github.com/puppetlabs/ca_extend) module designed for this
+task. Follow the module’s documentation to perform the extension in a supported
+and automated way.
```

Comment:
I think we are moving to "one phrase per line" it the markdown files, and the rest of the file seems to use it:

```suggestion
If you're using Bolt in your environment, there's a [ca_extend](https://github.com/puppetlabs/ca_extend) module designed for this task.
Follow the module’s documentation to perform the extension in a supported and automated way.
```